apollo-core/src/test/java/com/ctrip/framework/foundation/internals/UtilsTest.java

```diff
@@ -0,0 +1,89 @@
+package com.ctrip.framework.foundation.internals;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author Josh Knopp(https://github.com/joshknopp)
```

Comment:
Please delete the author in javadoc, because your change will be show in git's commit.

Reply:
No problem - was trying to follow style guide, but I'm indifferent 😄